### ISSUE TITLE
how to split the dataset

### ISSUE THREAD

**Issue by Laphroaig-sc** [arumaekawa/dataset-distillation-with-attention-labels#2]
Hello,

How is the 'ag_news' dataset divided into validation and test data? Are the validation data and test data the same?

Hoping to get a reply. Thanks

**Comment by arumaekawa** [comment]
Thank you for your message,
As we mentioned in the paper, we did not use validation set (ag_news does not provide validation split).
But please note that we did not intense hyperparameter search or model selection according to the performance on the test set, which is why there is no need to care about overfitting to the test set.

**Comment by Laphroaig-sc** [comment]
I see : )
Thank you so much!
